### PR TITLE
chore: Add PR template

### PR DESCRIPTION
**Pull request by nikhil-varma**
## Describe the issue/change
At the moment, the repo can use a consistent PR template in order to help gain consistent information about each PR

## Describe the changes proposed/implemented in this PR
This PR adds a PR template under a `.github` folder 

## Please check which browsers were used for testing

- [ ] Chrome
- [ ] Safari (OSX)
- [ ] Safari (iOS)
- [ ] Firefox

### CODE REVIEW

**Comment by s-yadav** [comment]
@nikhil-varma Can we add something about adding specs for failing case which PR is solving (if applicable.)

**Comment by nikhil-varma** [comment]
@s-yadav Good idea! I pushed an update. Please have a look. Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Describe the issue/change
+
+## Describe specs for failing cases if this is an issue (If application)
```
Comment (s-yadav) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
## Describe specs for failing cases if this is an issue (If applicable)
```

**Review by s-yadav — APPROVED** [review]

**Comment by s-yadav** [comment]
Looks good. Can you squash your commits.

**Comment by nikhil-varma** [comment]
@s-yadav Thanks! I don't have access to merge this. Can you please merge it? 😄

**Pull request merged**